ci/builders/mac_host_engine.json

```diff
@@ -129,7 +129,10 @@
                 "cpu=x86"
             ],
             "dependencies": [
-                {"dependency": "goldctl", "version": "git_revision:3a77d0b12c697a840ca0c7705208e8622dc94603"}
+                {
+                    "dependency": "goldctl",
+                    "version": "git_revision:3a77d0b12c697a840ca0c7705208e8622dc94603"
+                }
             ],
             "gclient_custom_vars": {
```

Comment:
Not sure what the scope of the PR is, but maybe it would be good to fix all of these? I'm happy to follow-up if you're OOO the rest of the day.

Reply:
My grep command didn't catch the gclient_custom_variables `grep -r gclient ./`. Updated.